### ISSUE TITLE
[A11y]Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger.

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9300]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1656375

Environment Details:
Application Name: NuGet Service
#URL: https://www.nuget.org/account/ForgotPassword
Windows version: Windows 10
ChromiumEdge Version: 105.0.1343.42 (Official Build) (64-bit)

Repro Steps:
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
Launch the application URL: https://www.nuget.org/account/ForgotPassword
Run the Accessibility insights for web 
Issue: 
Flags elements whose role is none or presentation and which cause the role conflict resolution to trigger. (presentation-role-conflict - https://dequeuniversity.com/rules/axe/4.4/presentation-role-conflict?application=msftAI)

Target application: NuGet Gallery | Forgot Password - https://www.nuget.org/account/ForgotPassword

Element path: iframe[title="reCAPTCHA"]

Snippet: <iframe title="reCAPTCHA" src="https://www.google.com/recaptcha/api2/anchor?ar=1&amp;k=6Lf4P10UAAAAAPp-L5yBSChLOGUdU4ci187t7JCv&amp;co=aHR0cHM6Ly93d3cubnVnZXQub3JnOjQ0Mw..&amp;hl=en&amp;v=vP4jQKq0YJFzU6e21-BGy3GP&amp;size=invisible&amp;cb=ioqna7bs53lh" width="256" height="60" role="presentation" name="a-le5htsccnaz0" frameborder="0" scrolling="no" sandbox="allow-forms allow-popups allow-same-origin allow-scripts allow-top-navigation allow-modals allow-popups-to-escape-sandbox allow-storage-access-by-user-activation" tabindex="1">

Similar issue observed in following scenarios:
https://www.nuget.org/packages/Microsoft.Azure.Services.AppAuthentication/1.6.1/ReportAbuse

How to fix:
Fix all of the following:
  Element is not focusable.

Environment: Microsoft Edge version 106.0.1370.52

Notes:
This appears to complain about the fix we made for tab ordering with the recaptcha iFrame. We should discuss what the proper tab ordering is here.
